src/Card/README.md

```diff
@@ -293,7 +295,9 @@ Note that `Card.Footer` has a separate `orientation` prop which will override th
 
   return (
     <Card style={{ width: isExtraSmall ? "100%" : "40%" }}>
-      <Card.ImageCap 
+      <Card.ImageCap
+        skeletonHeight={140}
+        capLogoSkeletonHeight={41}
```

Comment:
If this `Card` isn't in a loading state (no `isLoading` prop), do we need the `skeletonHeight` and `capLogoSkeletonHeight` props here?